WordPressAuthenticator/GoogleSignIn/URL+GoogleSignIn.swift

```diff
@@ -18,9 +18,11 @@ extension URL {
             ("code_challenge_method", pkce.method.urlQueryParameterValue),
             ("redirect_uri", clientId.defaultRedirectURI),
             ("response_type", "code"),
-            // TODO: We might want to add some of these or them configurable
-            // See https://developers.google.com/identity/protocols/oauth2/scopes
-            ("scope", "openid https://www.googleapis.com/auth/userinfo.email https://www.googleapis.com/auth/userinfo.profile")
+            // The Google Sign-In SDK use "profile email", but we only need the user's email address.
```

Comment:
The scopes are reduced from `profile + email` (when using the GoogleSignIn SDK), to `email` (when using our own implementation). Are you sure this won't cause any issue in the backend? Like, (I'm purely making assupmtion here) maybe the backend will use Google account's username to create a WP account?